_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -20,7 +20,9 @@ To define a detector:
 
 1. On the **Security Analytics** home page or the **Detectors** page, choose **Create detector**.
 1. Give the detector a name and, optionally, a description. 
-1. In the **Data source** section, select one or more sources for the log data. Use an asterisk (*) to indicate a wildcard pattern. When selecting multiple data sources, their logs must be of the same type. We recommend creating separate detectors for different log types.
+1. In the **Data source** section, select one or more sources for the log data. Use an asterisk (*) to indicate a wildcard pattern. When selecting multiple data sources, their logs must be of the same type. We recommend creating separate detectors for different log types. 
+
+   Data sources support the use of [aliases]({{site.url}}{{site.baseurl}}/im-plugin/index-alias/). However, if you're configuring an alias as data source, it must be attached to a **Write** index alias, since Security Analytics only supports [data stream]({{site.url}}{{site.baseurl}}/im-plugin/data-streams/) for time-series generated data. Furthermore, when using an alias, ensure that your documents are ingesting through the alias and **not** the index(es) the alias was created for. 
```

Comment:
@jowg-amazon: Is this more accurate?

```suggestion
   Data sources support the use of [aliases]({{site.url}}{{site.baseurl}}/im-plugin/index-alias/). However, if you're configuring an alias as data source, it must be attached to a **Write** index alias, since all [data streams]({{site.url}}{{site.baseurl}}/im-plugin/data-streams/) require a write index for time-series generated data. Furthermore, when using an alias, ensure that your documents are ingesting through the alias and **not** the index(es) the alias was created for. 
```

Reply:
I think `since all [data streams]({{site.url}}{{site.baseurl}}/im-plugin/data-streams/) require a write index ` part might still be a little misleading? Aliases and data streams do not depend on each other. Aliases require a write index because security analytics detectors are optimized so they only detect on write indices. Data streams happen to not need this additional note because they have a write index configured by definition. Maybe we can keep these two concepts separate and just simply mention that security analytics also supports data streams as a data source?

Reply:
Let me know if this latest update works.

Reply:
LGTM!